rd-net/Test.Lifetimes/Lifetimes/LifetimeTest.cs

```diff
@@ -228,7 +229,20 @@ void LoggerHandler(LeveledMessage message)
           () => TestLogger.ExceptionLogger.Handlers -= LoggerHandler
           );
 
+
         var lifetimeDefinition = lifetime.CreateNested();
+
+        var def2 = lifetime.CreateNested();
```

Comment:
Please do `using var`. Otherwise, this test's failure may affect further ones.

Reply:
it is not necessary because parent `lifetime` will be terminated when the test is finished

Reply:
Ah alright